action/awsconfig.go

```diff
@@ -24,11 +25,12 @@ type Clusters struct {
 }
 
 type ClusterConfig struct {
-	Name      string `yaml:"name"`
-	Alias     string `yaml:"alias"`
-	AccountID string `yaml:"accountId"`
-	Role      string `yaml:"role"`
-	Region    string `yaml:"region"`
+	Name        string `yaml:"name"`
+	Alias       string `yaml:"alias"`
+	AccountID   string `yaml:"accountId"`
+	Role        string `yaml:"role"`
+	Region      string `yaml:"region"`
+	Destination string `yaml:"destination"`
```

Comment:
could you add a test case in awsconfig_test?

Reply:
@karlderkaefer I might need your help with tests, I pushed some changes

Reply:
the test complains with `getKey, but does have GetKey` so you try to update

Reply:
but we could pair tomorrow